module-bluetooth/Bluetooth/interface/profiles/HFP/HFP.cpp

```diff
@@ -471,9 +459,10 @@ namespace bluetooth
     }
     void HFP::HFPImpl::initializeCall() const noexcept
     {
-        if (!isAudioRouted) {
-            LOG_DEBUG("Phone number is unused in BT stack - mocking");
-            hfp_ag_outgoing_call_initiated("1234567");
+        if (!isCallInitialized && !isIncomingCall) {
```

Comment:
just askin, only incoming call is handled in this flow?

Reply:
It it used only to initiate the outgoing call, rest of the logic is in setupCall() method, which is executed after the call has been answered.